application/extensions/admin/grid/FloatingActionsWidget/assets/floatingActions.js

```
@@ -0,0 +1,370 @@
+                    $ajaxLoader.hide();
+                    if ($modal.data('keepopen') !== 'yes') {
+                        $modal.modal('hide');
+                    } else {
+                        $modalBody.empty().html(html);
+                        if (typeof syncMassiveActionResultsTableCaption === 'function') {
+                            syncMassiveActionResultsTableCaption($modal, $modalBody);
+                        }
+                    }
+                    if (onSuccess) {
+                        /* eslint-disable-next-line no-eval */
+                        var func = eval(onSuccess);
```

Biome (2.5.1)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)